buildspec.yml

```diff
@@ -33,12 +35,6 @@ phases:
       - echo "GITHUB_BRANCH=${GITHUB_BRANCH} GITHUB_COMMIT=${GITHUB_COMMIT} GITHUB_TAG=${GITHUB_TAG} DOCKER_IMG_TAG=${DOCKER_IMG_TAG}"
       - echo "Creating a clean environment"
       - make ci
-      - echo "Check code formatting"
-      - make ci-check-format
-      - echo "Linting code"
-      - make lint
```

Comment:
`ci-check-format` and linting make more sense before starting unit test and would leave it here.